manager/manager.go

```diff
@@ -349,13 +358,20 @@ func (m *Manager) ManageVolume(volumeID string) error {
 				}
 
 				if meta.NextIssuanceTime == nil || m.clock.Now().After(*meta.NextIssuanceTime) {
-					log.Info("Triggering new issuance")
-					if err := m.issue(volumeID); err != nil {
-						log.Error(err, "Failed to issue certificate")
-						// retry the request in 1 second time
-						// TODO: exponentially back-off
-						continue
-					}
+					wait.ExponentialBackoffWithContext(ctx, wait.Backoff{
+						Duration: time.Second * 2,
+						Factor:   2.0,
+						Jitter:   0.5,
+						Steps:    6,
+						Cap:      time.Minute,
```

Comment:
This says:

* 2s is the 'base' amount of time to back off
* multiply the previous value by 2 if it fails
* add a jitter of +/- 1s (half of the `duration`)
* 6 steps (this actually may need tweaking.. it may mean that it resets back to 0 after 6 'backoffs' - need to test)
* with a max time between calls of 1m

Reply:
suggestion: add your very good explanation as a comment on top of `wait.ExponentialBackoffWithContext` 👍👍

Reply:
I've just unheld this for merge, but +1 and I'll open a PR in a sec with this :)